### ISSUE TITLE
[v2.1] Minecraft crashed while typing /chunkcopy copy 

### ISSUE THREAD

**Issue by RinixGG** [TheCSDev/mc-chunk-copy#2]
[crash-2022-04-03_21.45.02-client.txt](https://github.com/TheCSDev/mc-chunk-copy/files/8405266/crash-2022-04-03_21.45.02-client.txt)

**Comment by TheCSDev** [comment]
Hello, thank you for submitting an issue.
I took a look at the crash report as well as the source code for the mod, and I managed to find out what is causing the issue. Apparently I forgot to implement an error handler for the command tab completions. The bug will be resolved in the next update, which I will publish soon. Until I release a patch for that bug, try resolving the issue by using the following methods:
- Make sure there are no files inside of the `.minecraft/mods/chunkCopy/savedChunks` directory, only folders.
- Make sure your operating system does not deny the mod access to that directory.
